### PR TITLE
Modernize ircd-hybrid protocol module. Add support for upcoming ircd-…

### PR DESCRIPTION
**Pull request by miwob**
…hybrid versions.

### CODE REVIEW

File: modules/protocol/hybrid.cpp [hunk 1]
```diff
@@ -408,32 +378,35 @@ struct IRCDMessagePong : IRCDMessage
...
 	{
 		/* Servers other than our immediate uplink are introduced via SID */
 		if (params[1] != "1")
 			return;
 
-		new Server(source.GetServer() == NULL ? Me : source.GetServer(), params[0], 1, params[2], UplinkSID);
+		if (params.size() == 5)
+			UplinkSID = params[2];
+
+		new Server(source.GetServer() == NULL ? Me : source.GetServer(), params[0], 1, params[params.size() - 1], UplinkSID);
```
Comment (SadieCat) [inline modules/protocol/hybrid.cpp:394]:
```suggestion
		new Server(source.GetServer() == NULL ? Me : source.GetServer(), params[0], 1, params.back(), UplinkSID);
```

**Comment by miwob** [comment]
ircd-hybrid does RHOST since early 2017. Minimum supported version would be 8.2.23 I think.
What sourceless commands are you talking about specifically? Servers can add/remove *LINEs since about half a decade now or something.

**Comment by Adam-** [comment]
I don't mind bumping the min version but currently the docs and the module claim to support 8.2.x and I think this would break that. Specifically I am talking about all of the ones in this diff.

**Comment by miwob** [comment]
I've bumped the min version mentioned in the module. The docs don't seem to mention a specific version at all from what I have seen.

**Comment by Adam-** [comment]
They do here https://github.com/anope/anope/blob/2.0/docs/README#L171

**Pull request closed without merging**